src/lambda/config/templates.ts

```diff
@@ -5,11 +5,13 @@
 
 'use strict'
 
-import { parse, ParseError, ParseErrorCode } from 'jsonc-parser'
+// Use jsonc-parser.parse instead of JSON.parse, as JSONC can handle comments. VS Code uses jsonc-parser
+// under the hood to provide symbols for JSON documents, so this will keep us consistent with VS code.
```

Comment:
Not sure about others, but my computer redlines comments in `handlers.json`...do we want to support this?
![json](https://user-images.githubusercontent.com/29374703/54722266-5d21a200-4b21-11e9-8b9f-905a612e547f.png)

Reply:
If you change the "type" of the file to JSON with Comments, the squiggles go away

Reply:
Good catch, this is potentially confusing to users. If I open one of VS Code's config files (i.e. `.vscode/launch.json`, it defaults to "JSON with Comments". We should (in a separate change) find out how to configure the default file type, and set it to `jsonc`.

Reply:
https://github.com/aws/aws-toolkit-vscode/issues/390